docs/File-Formats.md

```diff
@@ -275,12 +276,19 @@ The Clinical Data Dictionary from MSKCC is used to normalize clinical data, and
 ## Discrete Copy Number Data
 The discrete copy number data file contain values that would be derived from copy-number analysis algorithms like [GISTIC 2.0](https://www.ncbi.nlm.nih.gov/sites/entrez?term=18077431) or [RAE](https://www.ncbi.nlm.nih.gov/sites/entrez?term=18784837). GISTIC 2.0 can be [installed](https://www.broadinstitute.org/cgi-bin/cancer/publications/pub_paper.cgi?mode=view&paper_id=216&p=t) or run online using the GISTIC 2.0 module on [GenePattern](https://cloud.genepattern.org). For some help on using GISTIC 2.0, check the [Data Loading: Tips and Best Practices](Data-Loading-Tips-and-Best-Practices.md) page. When loading case list data, the `_cna` case list is required. See the [case list section](#case-lists).
 
-### Meta file 
+### Wide vs Long format
+For CNA data two formats are supported: the so-called wide and long format.
+THe wide format can be described as a matrix with each row representing a genes and each column a sample.
+The long format has a row for each gene-sample combination, making the file a lot longer.
```

Comment:
```suggestion
For CNA data two formats are supported: the wide, and the long format:

- **Wide format**: a matrix, where each row is a gene, and each column is a sample
- **Long format**: not a matrix, each row is a gene-sample combination; this makes the file longer
```

Reply:
Fixed in [7e7ca7](https://github.com/cBioPortal/cbioportal/compare/646069c114345e4c1554f618c23396094122f4a0..7e7ca798ae5ac8bb041df89cfe1883252da1e964)